Concrete_article_data.py

```diff
@@ -0,0 +1,54 @@
+# -*-encoding: utf-8 -
+
+
+import numpy as np
+import pandas as pd
+from sklearn import gaussian_process
```

Comment:
When uisng `as`, we generally use a short alias name for the module... For example, `gp`.